tfhe/src/c_api/shortint/server_key/pbs.rs

```diff
@@ -114,55 +114,3 @@ pub unsafe extern "C" fn shortint_server_key_generate_bivariate_pbs_lookup_table
         *result = Box::into_raw(heap_allocated_lookup_table);
     })
 }
-
-#[no_mangle]
-pub unsafe extern "C" fn shortint_server_key_bivariate_programmable_bootstrap(
-    server_key: *const ShortintServerKey,
```

Comment:
Because IMO `smart_apply_lookup_table_bivariate` should be replaced by `smart_evaluate_bivariate_function`

The reason is that `smart_evaluate_bivariate_function` chooses the element to scale and the scale dynamically while with `smart_apply_lookup_table_bivariate` the lookup table is already built so these choices not possible which greatly limits the smartness of the operation

Reply:
agreed on the reduced smartness, disagree on removing the option with the LUT as it's giving finer control over the LUT application

Reply:
I renamed `smart_apply_lookup_table_bivariate` `apply_lookup_table_bivariate`